src/components/Tools/AdvisorCard.vue

```diff
@@ -53,7 +53,9 @@ export default defineComponent({
         ...this.onboardingData.minor
           .map(acronym => getAdvisor(acronym, 'minor', this.userInfo))
           .flat(),
```

Comment:
```suggestion
          .flatMap(acronym => getAdvisor(acronym, 'minor', this.userInfo))
```

Reply:
this was there before, but sure i'll change it